src/grt/grt-wave_options-parse.ads

```diff
@@ -0,0 +1,64 @@
+--  GHDL Run Time (GRT) - mono-thread version.
+--  Copyright (C) 2005 - 2014 Tristan Gingold
```

Comment:
Actually since I made some modifications in other packages (like Grt.Strings and Grt.Waves), should I also add my name to the copyright notice of the corresponding files ?